src/chi/javascript/components/modal.js

```diff
@@ -110,9 +111,24 @@ class Modal extends Component {
       return null;
     }
   }
+  
+  getElementHeight(el) {
+    let height;
+    const clone = el.cloneNode(true);
+
+    document.body.append(clone);
+    height = parseInt(window.getComputedStyle(clone).getPropertyValue('height').replace("px", ""));
+    clone.remove();
+    
+    return height;
+  }
```

Comment:
An element in the Dom, cloned at the end of the document body might have different height, as the width might be different. Please, use Util:calculateExternalHeight or any equivalent solution.